tests/test_all_pipeline.py

```diff
@@ -110,7 +110,10 @@ def train():
 
     # model initiaiton
     model = init_instance_by_config(task["model"])
+    print(model)
```

Comment:
Could  you remove these lines which seems  for debug?

Reply:
This is to test __repr__, I will add some comments.